doc/src/sphinx/util-cookbook/futures.rst

```diff
@@ -41,7 +41,7 @@ Scala:
 
   /** Convert from a Scala Future to a Twitter Future */
   implicit class RichScalaFuture[A](val sf: ScalaFuture[A]) extends AnyVal {
-    def asTwitter(implicit e: ExecutionContext): TwitterFuture[A] = {
+    def asTwitter(): TwitterFuture[A] = {
       val promise: TwitterPromise[A] = new TwitterPromise[A]()
       sf.onComplete {
```

Comment:
I think the implicit ExecutionContext is still needed here for `onComplete`.
https://github.com/scala/scala/blob/2.13.x/src/library/scala/concurrent/Future.scala#L121
Do you mind double checking if this is true?

Reply:
You're right; `asTwitter` still needs to pass in an implicit ExecutionContext since `onComplete` takes in two parameters: a Try function and an implicit ExecutionContext. 

Thanks for catching this! I'm still new to Scala, and I've learned a lot about implicit parameters tonight! 😃